Makefile

```diff
@@ -1,28 +1,35 @@
-.PHONY: dist plugin-data backend-plugin-data frontend-plugin-data
-
-dist: plugin-data
+.PHONY: dist 
+dist: #plugin-data
 	rm -rf ./dist
-	cd site && hugo --destination ../dist/html
+	hugo -s site --destination ../dist/html
```

Comment:
Should this now be `--destination dist/html`?

Reply:
To my own surprise: No. `destination` is relative to `-s`.

Reply:
To be clear: I've tested the changes and they generate the files in the correct place.